streaming/java/streaming-runtime/src/main/java/io/ray/streaming/runtime/util/EnvUtil.java

```diff
@@ -3,13 +3,29 @@
 import io.ray.runtime.RayNativeRuntime;
 import io.ray.runtime.util.JniUtils;
 import java.lang.management.ManagementFactory;
+import java.net.InetAddress;
+import java.net.UnknownHostException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 public class EnvUtil {
 
+  private static final Logger LOG = LoggerFactory.getLogger(EnvUtil.class);
+
   public static String getJvmPid() {
     return ManagementFactory.getRuntimeMXBean().getName().split("@")[0];
   }
 
+  public static String getHostName() {
```

Comment:
duplicated with `io.ray.runtime.util.NetworkUtil.getIpAddress(null)`

Reply:
This method is used to get hostname but not IpAddress.